src/core/lib/security/credentials/composite/composite_credentials.cc

```diff
@@ -130,12 +92,12 @@ bool grpc_composite_call_credentials::get_request_metadata(
   ctx = grpc_core::New<grpc_composite_call_credentials_metadata_context>(
       this, pollent, auth_md_context, md_array, on_request_metadata);
   bool synchronous = true;
-  const grpc_call_credentials_array& inner = ctx->composite_creds->inner();
+  const grpc_composite_call_credentials::CallCredentialsList& inner =
```

Comment:
I believe `grpc_composite_call_credentials::` is not needed here.

Reply:
Done.